guides/v2.2/extension-dev-guide/versioning/codebase-changes.md

```diff
@@ -86,19 +86,12 @@ Any change not listed below is considered a PATCH level change.
 | | Changed format of the returned method result | MAJOR|
 | | New method exception (excluding subtypes of existing exceptions) | MAJOR|
 | | New method exception (subtypes of an existing one) | PATCH|
-| **JavaScript Interface** (marked with `@api`) | New interface| MINOR|
-| | New method added | MINOR|
-| | Interface removed| MAJOR|
-| | Method removed | MAJOR|
-| | New required method argument | MAJOR|
-| | Changed method signature | MAJOR|
-| | Last argument added| MINOR|
-| **JavaScript class** (marked with `@api`| New class| MINOR|
+| **JavaScript class** (marked with `@api`)| New class| MINOR|
 | | New method added | MINOR|
+| | New argument added | MINOR|
 | | Class removed| MAJOR|
 | | Method removed | MAJOR|
-| | New required method argument | MAJOR|
```

Comment:
Why this is removed?

Reply:
Default argument is a ES6 feature that is not supported/used in core which makes required argument a blurry subject. In order for JS to check if an argument is required, you have to make an explicit check for it.

Reply:
Thanks for your explanation @michaelyu0123 , now is clear for me.